src/python/foglamp/core/service_registry/instance.py

```diff
@@ -97,8 +98,8 @@ def register(cls, name, s_type, address, port, protocol='http'):
                 raise Service.AlreadyExistsWithTheSameAddressAndPort
             if not isinstance(port, int):
                 raise Service.NonNumericPortError
-            if int(port) == 8082:
-                raise Service.ReservedPortError
+            # if int(port) == 8082:
+            #     raise Service.ReservedPortError
```

Comment:
we need to have specifications for reserved port. I think this is the core itself? so need to reserve the port.

1) core
2) UI
3) storage
4) admin + user api
5) device api (I assume just COAP/ COAPs only)
6) .. ?